src/main/java/com/conveyal/gtfs/error/GTFSError.java

```diff
@@ -50,10 +53,6 @@ public final String getErrorCode () {
         return this.getClass().getSimpleName();
     }
 
-    public Priority getPriority () {
-        return Priority.UNKNOWN;
-    }
```

Comment:
The original design was for priorities to be returned by this method, which would be overridden on each subclass. This makes it clear that the priority is a characteristic of the class rather than each instance of the class. It also eliminates one field from each instance, which is a minor benefit from a storage perspective but still simpler. If we are depending on priorities being associated with classes rather than instances (in the UI for example) then I'd recommend we use the original approach.

Reply:
Updated to reflect the original design.